website/docs/reference/resource-properties/deprecation_date.md

```diff
@@ -28,5 +28,46 @@ models:
 
 ## Definition
 
-The deprecation date of the model in YAML DateTime format. 
+The deprecation date of the model is formatted as a date, optionally with a timezone offset. Supported RFC 3339 formats include:
+- `YYYY-MM-DD hh:mm:ss.sss±hh:mm`
+- `YYYY-MM-DD hh:mm:ss.sss`
+- `YYYY-MM-DD`
 
+When `deprecation_date` does not include an offset from UTC, then it is interpreted as being in the system time zone of the dbt execution environment.
+
+## Explanation
+
+### Purpose
+
+Declaring a `deprecation_date` for a dbt model provides a mechanism to communicate plans and timelines for long-term support and maintenance and to facilitate change management.
+
+Setting a `deprecation_date` works well in conjunction with other [model governance](/docs/collaborate/govern/about-model-governance) features like [model versions](/docs/collaborate/govern/model-versions), but can also be used independently from them.
+
+### Warning messages
+
+When a project references a model that's slated for deprecation or the deprecation date has passed, a warning is generated. If it's a versioned model, with a newer version available, then the warning says so. This added bit of cross-team communication, from producers to consumers, is an advantage of using dbt's built-in functionality around model versions to facilitate migrations.
+
+Additionally, [`WARN_ERROR_OPTIONS`](/reference/global-configs/warnings) gives a mechanism whereby users can promote these warnings to actual runtime errors:
+- `DeprecatedModel` - warning when parsing a project that defines deprecated model(s)
+- `UpcomingDeprecationReference` - warning when referencing a model with a future deprecation date
+- `DeprecatedReference` - warning when referencing a model with a past deprecation date
```

Comment:
Maybe some clarification that:
- `DeprecatedModel` is for the _producer_ of the deprecated model
- `UpcomingDeprecationReference` and `DeprecatedReference` are for the _consumers_ of the deprecated model (could be in a different project!)